### ISSUE TITLE
Resolving JS/CSS files from fragment

### ISSUE THREAD

**Issue by fbrandel** [acesinc/convergent-ui#11]
I see that the `src` attribute of an image tag gets automatically rewritten to use the full URL to the "remote" service.

For `<script>` and `<link>` tags this is not the case. Is there a reason for that? 

As pointed out in the documentation, scripts and css files could be loaded with the `/cui-req://`prefix, but in the remote service I just want to include the scripts/css with a relative path starting with '/'.

So I think it would be great if these relative paths also get rewritten/prefixed with the services URL.

**Comment by andrewserff** [comment]
Yes, I see how that could be helpful. Our use case didn't need that as really, we weren't ever using the UI in the remote microservice by itself. So we always referenced the javascript with the `/cui-req` in our main ui service. I'll see if I can get it to scrape the `script` and `link` tags as well. If you have time to do it, please do and submit a PR as I'm not sure when I'll get to it.  But thanks pointing this out!
